.github/workflows/release-ci.yaml

```diff
@@ -78,7 +78,10 @@ jobs:
       - name: Patch RELEASED_VERSIONS
         env:
           STACKROX_TAG: ${{ github.ref_name }}
-          GITHUB_TOKEN: ${{ secrets.ROBOT_ROX_GITHUB_TOKEN }}
+          RHACS_BOT_GITHUB_USERNAME: ${{ secrets.RHACS_BOT_GITHUB_USERNAME }}
```

Comment:
Instead of adding new variables, have you considered getting the email and username through GH API (https://docs.github.com/en/free-pro-team@latest/rest/users/users?apiVersion=2022-11-28#get-the-authenticated-user)?

That would make the next rotation or switch easier.
Also, we have used `GITHUB_TOKEN` consistently for a while, so it would be sad to see it go :(

Reply:
> Instead of adding new variables, have you considered getting the email and username through GH API (https://docs.github.com/en/free-pro-team@latest/rest/users/users?apiVersion=2022-11-28#get-the-authenticated-user)?
> 
> That would make the next rotation or switch easier. 

We've had issues with GH rate limiting in the past, so while its worth considering, I'd avoid for this PR.

> Also, we have used `GITHUB_TOKEN` consistently for a while, so it would be sad to see it go :(

This made me think about the env names in general. Having the secrets with `RHACS_BOT_` prefix makes sense but the code should use generic names.